storage/include/storage/db_interface.h

```diff
@@ -76,6 +77,7 @@ class IDBClient {
   virtual std::string getPath() const { return ""; };
   virtual void setCheckpointPath(const std::string&){};
   virtual std::string getCheckpointPath() const { return "NotSupported"; };
+  virtual std::string getPathForCheckpoint(std::uint64_t checkpointId) const { ConcordAssert(false); };
```

Comment:
checkpoint_id

Reply:
You are right, but I have intentionally kept camelCase, because this whole file uses it. I think it would be better to fix it for the whole file in a separate commit.